--- "a/\346\210\221\350\246\201\351\240\230\345\217\227\344\270\273_\351\240\214\346\201\251458_\345\271\273\347\207\210.jianpu.ily"	2017-02-07 12:20:20.272561691 -0700
+++ "b/\346\210\221\350\246\201\351\240\230\345\217\227\344\270\273_\351\240\214\346\201\251458_\345\271\273\347\207\210.jianpu.ily"	2017-02-07 11:51:57.554451227 -0700
@@ -201,7 +201,7 @@
   \applyOutput #'Voice #note-two d'8]
 )
 \once \override Tie #'transparent = ##t \once \override Tie #'staff-position = #0 |
-  \applyOutput #'Voice #note-one c'4[ ~
+  \applyOutput #'Voice #note-one c'4 ~
 \once \override Tie #'transparent = ##t \once \override Tie #'staff-position = #0 #(define (note-dashone grob grob-origin context)
   (if (grob::has-interface grob 'note-head-interface)
     (begin
@@ -210,7 +210,7 @@
           (make-lower-markup 0.5 (make-bold-markup "–")))))))
   \applyOutput #'Voice #note-dashone c'4 ~
   \applyOutput #'Voice #note-dashone c'4
-  \applyOutput #'Voice #note-five g4]-\tweak #'Y-offset #-1.2 -\tweak #'X-offset #0.6 _.
+  \applyOutput #'Voice #note-five g4-\tweak #'Y-offset #-1.2 -\tweak #'X-offset #0.6 _.
 |
   \applyOutput #'Voice #note-three e'4[
   \applyOutput #'Voice #note-one c'4
@@ -328,10 +328,10 @@
   \applyOutput #'Voice #note-two d'8]
 )
 \once \override Tie #'transparent = ##t \once \override Tie #'staff-position = #0 |
-  \applyOutput #'Voice #note-one c'4[ ~
+  \applyOutput #'Voice #note-one c'4 ~
 \once \override Tie #'transparent = ##t \once \override Tie #'staff-position = #0   \applyOutput #'Voice #note-dashone c'4 ~
   \applyOutput #'Voice #note-dashone c'4
-  \applyOutput #'Voice #note-five g4]-\tweak #'Y-offset #-1.2 -\tweak #'X-offset #0.6 _.
+  \applyOutput #'Voice #note-five g4-\tweak #'Y-offset #-1.2 -\tweak #'X-offset #0.6 _.
 |
   \applyOutput #'Voice #note-three e'4[
   \applyOutput #'Voice #note-one c'4
@@ -449,10 +449,10 @@
   \applyOutput #'Voice #note-two d'8]
 )
 \once \override Tie #'transparent = ##t \once \override Tie #'staff-position = #0 |
-  \applyOutput #'Voice #note-one c'4[ ~
+  \applyOutput #'Voice #note-one c'4 ~
 \once \override Tie #'transparent = ##t \once \override Tie #'staff-position = #0   \applyOutput #'Voice #note-dashone c'4 ~
   \applyOutput #'Voice #note-dashone c'4
-  \applyOutput #'Voice #note-five g4]-\tweak #'Y-offset #-1.2 -\tweak #'X-offset #0.6 _.
+  \applyOutput #'Voice #note-five g4-\tweak #'Y-offset #-1.2 -\tweak #'X-offset #0.6 _.
 |
   \applyOutput #'Voice #note-three e'4[
   \applyOutput #'Voice #note-one c'4
@@ -570,10 +570,10 @@
   \applyOutput #'Voice #note-two d'8]
 )
 \once \override Tie #'transparent = ##t \once \override Tie #'staff-position = #0 |
-  \applyOutput #'Voice #note-one c'4[ ~
+  \applyOutput #'Voice #note-one c'4 ~
 \once \override Tie #'transparent = ##t \once \override Tie #'staff-position = #0   \applyOutput #'Voice #note-dashone c'4 ~
   \applyOutput #'Voice #note-dashone c'4
-  \applyOutput #'Voice #note-five g4]-\tweak #'Y-offset #-1.2 -\tweak #'X-offset #0.6 _.
+  \applyOutput #'Voice #note-five g4-\tweak #'Y-offset #-1.2 -\tweak #'X-offset #0.6 _.
 |
   \applyOutput #'Voice #note-three e'4[
   \applyOutput #'Voice #note-one c'4
